source/stylesheets/refills/_progress-bar-indication.scss

```diff
@@ -1,45 +1,50 @@
-.progress-bar-indication {  
-  $base-border-color: gainsboro !default;
-  $base-border-radius: 3px !default;
-  $base-background-color: white !default;
-  $base-line-height: 1.5em !default;
-  $action-color: #477DCA !default;
-  $progress-border-color: $base-border-color;
-  $progress-border: 1px solid $progress-border-color;
-  $progress-meter-border-color: $action-color;
-  $progress-meter-border: 1px solid darken($progress-meter-border-color, 15%);
-  $progress-meter-color: $progress-meter-border-color;
-  $progress-background: darken($base-background-color, 5);
-  $progress-animation-duration: 0.7s;
-  $progress-color: white;
+.progress-bar-with-status {
+  $base-background-color: #fff !default;
+  $base-accent-color: #477dca !default;
+  $progress-background-color: darken($base-background-color, 5);
+  $progress-height: 24px;
+  $progress-status-color: $base-accent-color;
+  $progress-width: 100%;
+  $progress-border-radius: $progress-height;
 
-  background-color: $progress-background;
-  border-radius: $base-border-radius;
-  border: $progress-border;
-  box-shadow: inset 0 0 3px 0 rgba(darken($progress-background, 50%), 0.15);
-  margin: 0 auto;
-  width: 100%;
+  position: relative;
 
-  > span.meter {
-    @include box-sizing(border-box);
-    background-color: $progress-meter-color;
-    background-repeat: repeat-x;
-    background-size: 40px 40px;
-    border-bottom-right-radius: 0;
-    border-radius: $base-border-radius /1.5;
-    border-top-right-radius: 0;
-    border: $progress-meter-border;
+  .progress-bar-indication {
+    @include appearance(none);
+    background-color: $progress-background-color;
+    border: none;
+    border-radius: $progress-border-radius;
+    color: $progress-status-color;
     display: block;
-    height: 100%;
-    width: 60%;
+    height: $progress-height;
+    width: $progress-width;
+
+    &::-webkit-progress-bar {
+      background-color: $progress-background-color;
+      border-radius: $progress-border-radius;
+    }
+
+    &::-webkit-progress-value {
+      background-color: $progress-status-color;
+      border-radius: $progress-border-radius;
+    }
+
+    &::-moz-progress-bar {
+      background-color: $progress-status-color;
+      border-radius: $progress-border-radius;
+    }
+
+    &::-ms-fill {
+      background-color: $progress-status-color;
+      border: none;
+    }
   }
 
-  p {
-    color: $progress-color;    
+  .progress-bar-status {
+    @include position(absolute, 50% null null 1em);
+    @include transform(translateY(-50%));
+    color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.